cmd/nvidia-dra-plugin/nvlib.go

```diff
@@ -498,3 +512,21 @@ func (l deviceLib) setComputeMode(uuids []string, mode string) error {
 	}
 	return nil
 }
+
+// detactSupportTimeSliceByArch Determine whether the architecture series
+// supports setting time slices based on the gpu architecture.
+func detactSupportTimeSliceByArch(arch string) bool {
+	// todo: More information is needed to determine the support of various architectures
+	switch arch {
+	case "Pascal":
+		return false
+	case "Ada":
+	case "Ampere":
+	case "Turing":
+	case "Hopper":
+		return true
+	default:
+		return false
+	}
+	return false
```

Comment:
Note: it is unlikely that newer generations of GPUs will *NOT* support timeslicing. I would return `true` by default and only override this for architectures / scenarios where we know that timeslicing is not supported.

Reply:
It's probably better to use the CUDA compute capability so we can just check a number. I believe time-slicing is available on Volta+ architectures, so the check would simply be `cudaComputeCapability >= 7.0`.